pages/agent/v3/ssh_keys.md.erb

```diff
@@ -123,3 +123,19 @@ set -eu
 
 export SSH_AUTH_SOCK="/var/lib/buildkite-agent/.ssh/ssh-agent.sock"
 ```
+
+## Using multiple keys by adding pipeline-specific environments  
+
+There is yet another way to use multiple SSH keys.  
+
+1. Add a pipeline-specific environment (for example, by using [Elastic Stack's secrets support](https://github.com/buildkite/elastic-ci-stack-for-aws#build-secrets) or by having an Agent environment hook that switches on the repository URL or the pipeline slug):  
+```bash
+GIT_SSH_COMMAND="ssh -i ~/.ssh/id_rsa_mypipeline"
+```
+2. Create an identity file at that location:
+```bash
+~/.ssh/id_rsa_mypipeline
+```
+3. Add the public key for that identity file to `mypipeline` on the git repository provider
+
+If you are using Elastic Stack, this approach is redundant as [secrets support](https://github.com/buildkite/elastic-ci-stack-for-aws#build-secrets) allows you to specify an SSH key per pipeline as `/{pipeline-slug}/private_ssh_key`. 
```

Comment:
And let's preload this *before* the list of instructions.

Reply:
Done.